consensus/dbft/dbft_service.go

```diff
@@ -135,12 +136,12 @@ func (this *DbftService) GetPID() *actor.PID {
 	return this.pid
 }
 func (this *DbftService) Start() error {
-	this.pid.Tell(&actorTypes.StartConsensus{})
+	this.pid.Tell(&actormsg.StartConsensus{})
```

Comment:
start/stop consensus module asynchronously with actor, may make consensus module miss some message.
suggest not use this interface.

Reply:
or change starting/stopping consensus actor to be synchronous

Reply:
ok.now use  synchronization method  operator starting/stopping consensus. use RequestFuture method.